content/docs/1.18/opentelemetry.md

```diff
@@ -0,0 +1,115 @@
+---
+title: OpenTelemetry
+hasparent: true
+---
+
+The future Jaeger backend components will be based on [OpenTelemetry collector](https://opentelemetry.io/docs/collector/about/).
+This integration will make all OpenTelemetry Collector features available in the Jaeger backend components.
+
+{{< warning >}}
+At the moment Jaeger OpenTelemetry components are experimental and the configuration and behavior can change.
+The current progress can be tracked via [issues tagged as `area/otel`](https://github.com/jaegertracing/jaeger/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%2Fotel).
+{{< /warning >}}
+
+The Jaeger OpenTelemetry backend components are published as Docker images:
+
+Component             | Repository
+--------------------- | ---
+**jaeger-agent**      | [hub.docker.com/r/jaegertracing/jaeger-opentelemetry-agent/](https://hub.docker.com/r/jaegertracing/jaeger-opentelemetry-agent/)
+**jaeger-collector**  | [hub.docker.com/r/jaegertracing/jaeger-opentelemetry-collector/](https://hub.docker.com/r/jaegertracing/jaeger-opentelemetry-collector/)
+**jaeger-ingester**   | [hub.docker.com/r/jaegertracing/jaeger-opentelemetry-ingester/](https://hub.docker.com/r/jaegertracing/jaeger-opentelemetry-ingester/)
+**all-in-one**        | [hub.docker.com/r/jaegertracing/opentelemetry-all-in-one/](https://hub.docker.com/r/jaegertracing/opentelemetry-all-in-one/)
+
+## Compatibility
+
+The Jaeger OpenTelemetry components are **almost** backward compatible legacy Jaeger components.
+
+The differences are:
+
+* Health check port changed to `13133`
+* Not all legacy Jaeger flags are exposed (e.g. health check port)
+* Exposed metrics
+
+## Configuration
+
+Jaeger OpenTelemetry components can be configured by a subset of Jaeger legacy flags (or other [configuration sources](../cli/))
+and [OpenTelemetry configuration file](https://opentelemetry.io/docs/collector/configuration/).
+The OpenTelemetry configuration takes precedence over Jaeger configuration.
+The Jaeger OpenTelemetry components use hardcoded default configuration that enables predefined set of components - Jaeger receiver, attribute processor, (storage) exporter.
```

Comment:
Components is used here in two different contexts: "Jaeger OpenTelemetry components" i.e agent, collector, ingester, and "predefined set of components" - meaning the exporters, processors, receivers. Would be good if we could change one?

Reply:
+1 first changed to binaries